metadata.json

```diff
@@ -8,9 +8,9 @@
     "summary": "Resource types for managing settings in HOCON files",
     "operatingsystem_support": [
       {
-        "operatingsystem": "CentOS",
+        "operatingsystem": "Debian",
```

Comment:
I changed this, as we no longer have any CentOS nodesets in our acceptance test directory. All the nodesets we had (including CentOS 6) didn't work because they contained Ruby 1.8.

I think the plan is to get more acceptance nodesets and possibly Ruby 1.8 support before we do a 1.0 release.